.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -8,33 +8,18 @@ assignees: ''
 ---
 
 **Describe and demonstrate the bug**
-Please attach a [ucm transcript](https://www.unison-lang.org/docs/tooling/transcripts/) if possible, calling out the unexpected behavior in the text. e.g.
+This should be written as a [ucm transcript](https://www.unison-lang.org/docs/tooling/transcripts/) if possible, calling out the unexpected behavior in the text. e.g.
 
-Input:
-```` markdown
-``` unison :hide
-a = 1
-```
-Here I typo the next command and `ucm` silently does nothing. I would have expected an error message:
-``` ucm
-.> add b
-```
-````
-
-Output:
-```` markdown
-``` unison
+``` unison :hidec
```

Comment:
Ok, I have now added transcript tests for other things in the repo that are transcript-y. That includes this bug-report template and things in docs/.

Adding those, I found a bug in the transcript parser, where it was a bit optimistic, and `unison :hidec` got parsed as `unison :hide c` rather than failing with an invalid tag, so fixed that as well.